internal/connector/internal/api/admin/private/api/openapi.yaml

```diff
@@ -266,14 +266,14 @@ paths:
       summary: Get a list of available connector namespaces in cluster
       tags:
       - Connector Clusters Admin
-  /api/connector_mgmt/v1/admin/kafka_connector_namespaces/${namespace_id}/connectors:
+  /api/connector_mgmt/v1/admin/kafka_connector_clusters/${connector_cluster_id}/connectors:
```

Comment:
That $ sign there feels odd, not sure it should be there?

Reply:
yeah, it's a typo, will remove. Good catch. It's interesting that openapi generator and validation doesn't complain. I guess it expects a `$` sign in the url in this case??

Reply:
yep

Reply:
I've opened an issue for that https://issues.redhat.com/browse/MGDCTRS-836
The opening generator does not complain as it interprets the '$' as is so it's just a char (but fails the invocation of endpoints as the IDs are then not found)